### PR TITLE
Adds more descriptions to missing items and changes some

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

Adds more descriptions to missing items and changes some to be better.

## Why It's Good For The Game
Empty examines is bad

## Testing

Compiled, looked at some stuff.

## Changelog
:cl:
add: Several objects missing descriptions have been given descriptions
/:cl:

### CODE REVIEW

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -48,6 +48,7 @@ GLOBAL_LIST_EMPTY(airlock_emissive_underlays)
 
 /obj/machinery/door/airlock
 	name = "airlock"
+	desc = "An airlock door keeping you safe from the vacuum of safe. Only works if closed."
```
Comment (Fordoxia) [inline code/game/machinery/doors/airlock.dm:51]:
```suggestion
	desc = "An airlock door keeping you safe from the vacuum of space. Only works if closed."
```

File: code/game/machinery/doors/airlock_types.dm [hunk 2]
```diff
@@ -540,7 +540,7 @@
 	glass = stealth_glass
 	airlock_material = stealth_airlock_material
 	name = "airlock"
-	desc = "It opens and closes."
+	desc = "An airlock door keeping you safe from the vacuum of safe. Only works if closed."
```
Comment (Fordoxia) [inline code/game/machinery/doors/airlock_types.dm:543]:
```suggestion
	desc = "An airlock door keeping you safe from the vacuum of space. Only works if closed."
```

File: code/game/machinery/doors/windowdoor.dm [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /obj/machinery/door/window
 	name = "interior door"
-	desc = "A strong door."
+	desc = "A door thats also a window, isn't that amazing."
```
Comment (Fordoxia) [inline code/game/machinery/doors/windowdoor.dm:3]:
```suggestion
	desc = "A door that's also a window, isn't that amazing?"
```

File: code/game/machinery/doors/windowdoor.dm [hunk 4]
```diff
@@ -435,8 +436,8 @@
 	base_state = "right"
 
 /obj/machinery/door/window/antitheft
-	desc = "Better do not try to steal, pal, lube off."
 	name = "Branch, do not add stuff here"
+	desc = "Better do not try to steal, pal, lube off."
```
Comment (Fordoxia) [inline code/game/machinery/doors/windowdoor.dm:440]:
```suggestion
	desc = "Better not try to steal, pal, lube off."
```

File: code/game/machinery/iv_drip.dm [hunk 5]
```diff
@@ -3,6 +3,7 @@
 
 /obj/machinery/iv_drip
 	name = "\improper IV drip"
+	desc = "Simply attach a bloodbag, and stab the patient, and they'll have more blood in no time."
```
Comment (Fordoxia) [inline code/game/machinery/iv_drip.dm:6]:
```suggestion
	desc = "Simply attach a blood bag, and stab the patient, and they'll have more blood in no time. You can put other stuff in the bags too, I guess."
```

File: code/modules/paperwork/photocopier.dm [hunk 6]
```diff
@@ -4,6 +4,7 @@
 
 /obj/machinery/photocopier
 	name = "photocopier"
+	desc = "For making copies of important documents, or more likely your ass."
```
Comment (Fordoxia) [inline code/modules/paperwork/photocopier.dm:7]:
```suggestion
	desc = "For making copies of important documents, or more likely, your ass."
```

File: code/modules/power/lights.dm [hunk 7]
```diff
@@ -164,17 +164,17 @@
   */
 /obj/machinery/light
 	name = "light fixture"
+	desc = "Yep, its a light."
```
Comment (Fordoxia) [inline code/modules/power/lights.dm:167]: Technically it IS actually a fixture, not a light. The light is attached to it.

Comment (1080pCat) [reply code/modules/power/lights.dm:167]: ☝️🤓

File: code/modules/reagents/chemistry/machinery/chem_master.dm [hunk 8]
```diff
@@ -608,6 +609,7 @@
 
 /obj/machinery/chem_master/condimaster
 	name = "\improper CondiMaster 3000"
+	desc = "Used by remove reagents from that single beaker you're using. Or create condiment packs and bottles, your choice."
```
Comment (Christasmurf) [inline code/modules/reagents/chemistry/machinery/chem_master.dm:612]:
```suggestion
	desc = "Used to remove reagents from that single beaker you're using. Or create condiment packs and bottles, your choice."
```

File: code/game/machinery/doors/windowdoor.dm (hunk 4 above)
Comment (Contrabang) [reply code/game/machinery/doors/windowdoor.dm:440]: I was just moving this comment, and I think it is fine as it is.

File: code/game/machinery/iv_drip.dm (hunk 5 above)
Comment (Contrabang) [reply code/game/machinery/iv_drip.dm:6]: I feel like thats something for the blood bag itself

**Comment by Danchi299** [comment]
do we need description for floors?
those always make me feel empty when I miss-examine them

**Comment by Contrabang** [comment]
Not gonna touch floors currently, I think its fine they have no desc.

File: code/game/machinery/guestpass.dm [hunk 9]
```diff
@@ -33,6 +33,7 @@
 
 /obj/machinery/computer/guestpass
 	name = "guest pass terminal"
+	desc = "This console allows staff to give out temporary access to their coworkers."
```
Comment (FunnyMan3595) [inline code/game/machinery/guestpass.dm:36]:
```suggestion
	desc = "This console allows staff to give out temporary access to the clown."
```

File: code/game/machinery/guestpass.dm [hunk 10]
```diff
@@ -231,6 +232,7 @@
 
 /obj/machinery/computer/guestpass/hop
 	name = "\improper HoP guest pass terminal"
+	desc = "The Head of Personnel's guest pass terminal allows the HoP to give out any temporary access."
```
Comment (FunnyMan3595) [inline code/game/machinery/guestpass.dm:235]:
```suggestion
	desc = "The Head of Personnel's guest pass terminal allows the HoP to temporarily allow anyone into places they probably shouldn't be."
```

File: code/game/machinery/iv_drip.dm [hunk 11]
```diff
@@ -3,6 +3,7 @@
 
 /obj/machinery/iv_drip
 	name = "\improper IV drip"
+	desc = "Simply attach a bloodbag and puncture the patient with a needle, they'll have more blood in no time."
```
Comment (FunnyMan3595) [inline code/game/machinery/iv_drip.dm:6]:
```suggestion
	desc = "Simply attach a bloodbag and puncture the patient with a needle, they'll have more blood in no time. Or less."
```

File: code/game/objects/items/devices/radio/intercom.dm [hunk 12]
```diff
@@ -296,7 +296,7 @@
 
 /obj/item/radio/intercom/locked/prison
 	name = "prison intercom"
-	desc = "Talk through this. It looks like it has been modified to not broadcast."
+	desc = "A reliable form of communication even during local communication blackouts. It looks like it has been modified to not broadcast. Not so reliable I guess..."
```
Comment (FunnyMan3595) [inline code/game/objects/items/devices/radio/intercom.dm:299]:
```suggestion
	desc = "A reliable form of communication even during local communication blackouts. It looks like it has been modified to not broadcast. Not so reliable, I guess..."
```

File: code/game/objects/items/weapons/storage/fancy.dm [hunk 13]
```diff
@@ -38,6 +38,7 @@
 
 /obj/item/storage/fancy/donut_box
 	name = "donut box"
+	desc = "\"To do nut, or don't nut, the choice is obvious.\""
```
Comment (FunnyMan3595) [inline code/game/objects/items/weapons/storage/fancy.dm:41]:
```suggestion
	desc = "\"To do, or do nut, the choice is obvious.\""
```

File: code/modules/library/library_equipment.dm [hunk 14]
```diff
@@ -159,6 +159,7 @@
  */
 /obj/machinery/bookbinder
 	name = "Book Binder"
+	desc = "Used by authors, poets, and librarians to scan papers and print copies of their books."
```
Comment (FunnyMan3595) [inline code/modules/library/library_equipment.dm:162]:
```suggestion
	desc = "Used by authors, poets, and librarians to scan papers and print copies of their fanfics."
```

File: code/modules/power/lights.dm [hunk 15]
```diff
@@ -231,12 +231,12 @@
   */
 /obj/machinery/light/small
 	icon_state = "bulb1"
-	base_state = "bulb"
+	desc = "A compact and cheap light fixture, perfect for subtle illumination and maintenance tunnels."
```
Comment (FunnyMan3595) [inline code/modules/power/lights.dm:234]:
```suggestion
	desc = "A compact and cheap light fixture, perfect for keeping maintenance tunnels appropriately spooky."
```

File: code/modules/reagents/chemistry/machinery/chem_dispenser.dm [hunk 16]
```diff
@@ -1,5 +1,6 @@
 /obj/machinery/chem_dispenser
 	name = "chem dispenser"
+	desc = "For making chemicals, medicine, and bombs."
```
Comment (FunnyMan3595) [inline code/modules/reagents/chemistry/machinery/chem_dispenser.dm:3]:
```suggestion
	desc = "For making chemicals, medicine, and explosions."
```

"Medchem blew up." "Again?"

**Review by FunnyMan3595 — COMMENTED** [review]
A variety of silly suggestions, feel free to adopt, adapt, or ignore as you see fit.

File: code/game/machinery/guestpass.dm (hunk 9 above)
Comment (Contrabang) [reply code/game/machinery/guestpass.dm:36]: Going to keep the old one for this, i think its needed to portray what it does

**Review by Henri215 — APPROVED** [review]
Lewcs good to me

**Review by S34NW — APPROVED** [review]

**Pull request merged**